ooniprobe/View/RunTest/TestOverviewViewController.m

```diff
@@ -81,14 +78,14 @@ -(void)changeConstraints{
 -(void)reloadLastMeasurement{
     dispatch_async(dispatch_get_main_queue(), ^{
         [self.estimatedLabel setText:NSLocalizedString(@"Dashboard.Overview.Estimated", nil)];
-        [self.estimatedDetailLabel setText:
-         [NSString stringWithFormat:@"%@ %@",
-          testSuite.dataUsage,
-          [LocalizationUtility getReadableRuntime:[testSuite getRuntime]]]];
+//        [self.estimatedDetailLabel setText:
+//         [NSString stringWithFormat:@"%@ %@",
+//          testSuite.dataUsage,
+//          [LocalizationUtility getReadableRuntime:[testSuite getRuntime]]]];
```

Comment:
Is this code that we still need to convert or code that we can safely get rid with?

Reply:
Thanks for pointing this out. I need to convert it to something else. Thats why its commented not deleted

Reply:
Thanks!